outlookmail/utils.bal

```diff
@@ -113,30 +113,32 @@ isolated function addOdataFileType(MessageContent|FileAttachment messageContent)
     } else {
         attachments = messageContent?.message?.attachments ?: [];
     }
-    foreach int i in 0 ... attachments.length() {
-        FileAttachment attachment = attachments.remove(0);
-        FileAttachment attachmentTemp = {
-            contentBytes: attachment.contentBytes,
-            name: attachment.name,
-            contentType: attachment.contentType,
-            "@odata.type": "#microsoft.graph.fileAttachment"
-        };
-        if (attachment?.id is string) {
-            attachmentTemp.id = attachment?.id.toString();
-        }
-        if (attachment?.contentId is string) {
-            attachmentTemp.contentId = attachment?.contentId.toString();
-        }
-        if (attachment?.isInline is boolean) {
-            attachmentTemp.isInline = <boolean>attachment?.isInline;
-        }
-        if (attachment?.lastModifiedDateTime is string) {
-            attachmentTemp.lastModifiedDateTime = attachment?.lastModifiedDateTime.toString();
-        }
-        if (attachment?.size is int) {
-            attachmentTemp.size = <int>attachment?.size;
+    if (attachments.length() > 0) {
+        foreach int i in 0 ... attachments.length() {
+            FileAttachment attachment = attachments.remove(0);
```

Comment:
We are not getting the i th element, rather we always get 0th. 
Any reason for that?